libraries/rush-lib/src/cli/RushXCommandLine.ts

```diff
@@ -207,6 +208,26 @@ export class RushXCommandLine {
       }
     });
 
+    if (rushConfiguration?.packageManager === 'pnpm' && rushConfiguration?.experimentsConfiguration) {
```

Comment:
Can this be moved into a shared plugin?

Reply:
What do you mean `a shared plugin`?  Is `rushx build` same as `rush build`? Which can take plugins as well?

Reply:
Good question. If it isn't pluggable, it should be. That can be done in a follow-up.

Reply:
In my opinion this feature should NOT be moved into a Rush plugin.  The implementation of `pnpm-sync` is lightweight enough to be part of core Rush.  And it would not be a good experience for a core operation like `rush install` to depend on autoinstallation of a plugin.

Reply:
I mean a built-in plugin. The idea would be to further develop the plugin API.